host/xtest/xtest_test.h

```diff
@@ -122,4 +122,20 @@ extern const TEEC_UUID socket_ta_uuid;
 extern const TEEC_UUID sdp_basic_ta_uuid;
 extern char *_device;
 
+#ifndef TA_DIR
+# ifdef __ANDROID__
+#define TA_DIR "/system/lib/optee_armtz"
+# else
+#define TA_DIR "/lib/optee_armtz"
+# endif
+#endif
+
+#ifndef TA_TEST_DIR
```

Comment:
Looks like `TA_TEST_DIR` is not used anymore

Reply:
It's still used in `host/xtest/regression_1000.c`. Should I move it back to `host/xtest/regression_1000.c` again?

Reply:
It's not used in the HEAD of this PR, right?

```
$ git fetch origin pull/240/head
From https://github.com/OP-TEE/optee_test
 * branch                      refs/pull/240/head -> FETCH_HEAD
$ git checkout FETCH_HEAD 
HEAD is now at 80b8e5941579... [review] xtest: add --install-tas argument
$ git grep TA_TEST_DIR
host/xtest/xtest_test.h:#ifndef TA_TEST_DIR
host/xtest/xtest_test.h:#  define TA_TEST_DIR "/data/tee/optee_armtz"
host/xtest/xtest_test.h:#  define TA_TEST_DIR "/tmp/optee_armtz"
```

So I think it should remain in `host/xtest/regression_1000.c` until it finally gets removed by "regression 1008: test with corrupt BSTA".

Reply:
Aha, sorry I confused it with `TA_DIR`.